deployment/deployment.json

```diff
@@ -2832,7 +2832,7 @@
         "status": "dev",
         "versions": {
           "schema": "1.3.0",
-          "subgraph": "1.1.2",
+          "subgraph": "1.1.6",
```

Comment:
It's intentional, since 1.1.5 was already on `saddle-finance-ethereum`: https://github.com/messari/subgraphs/pull/1356/files#diff-8dffa45a56a9958c185c47bc9e8a84224a467f19c77ef8dfafde24cf6c8a6b39L2857